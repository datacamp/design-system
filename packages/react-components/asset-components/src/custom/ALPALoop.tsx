import React from 'react';

import { Alpa, Apply, Assess, Learn, Practice } from '../generated/alpa';

export const loopComponents = {
  Apply,
  Assess,
  Learn,
  Practice,
};

export type Highlight = keyof typeof loopComponents;

interface ALPALoopProps extends React.ComponentProps<typeof Alpa> {
  highlight?: Highlight;
}

const ALPALoop: React.FC<ALPALoopProps> = ({ highlight, ...otherProps }) => {
  const Logo = highlight ? loopComponents[highlight] : Alpa;
  return <Logo {...otherProps} />;
};

export default ALPALoop;
