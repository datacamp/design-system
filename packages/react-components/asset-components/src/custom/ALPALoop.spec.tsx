import axeRender from '@datacamp/waffles-axe-render';

import ALPALoop, { Highlight, loopComponents } from './ALPALoop';

describe('<ALPALoop />', () => {
  (Object.keys(loopComponents) as Highlight[]).forEach((highlight) => {
    it(`renders a ${highlight} logo`, async () => {
      const { container } = await axeRender(<ALPALoop highlight={highlight} />);
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
