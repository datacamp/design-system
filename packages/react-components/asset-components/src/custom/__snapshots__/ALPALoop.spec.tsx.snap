// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ALPALoop /> renders a Apply logo 1`] = `
<svg
  viewBox="0 0 674 670"
>
  <defs>
    <filter
      filterUnits="objectBoundingBox"
      height="100%"
      id="APPLY_svg__a"
      width="100%"
      x="0%"
      y="0%"
    >
      <feoffset
        in="SourceAlpha"
        result="shadowOffsetOuter1"
      />
      <fecolormatrix
        in="shadowOffsetOuter1"
        values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0"
      />
    </filter>
    <path
      d="M31.05 237.05l9.9 9.9L19.899 268l21.051 21.05-9.9 9.9L.101 268z"
      id="APPLY_svg__b"
    />
  </defs>
  <g
    fill="none"
    fill-rule="evenodd"
  >
    <g
      opacity="0.3"
      transform="translate(27)"
    >
      <path
        d="M310 24v14C157.987 38 32.566 151.435 14.29 297.998H.185C18.54 143.675 150.253 24 310 24z"
        fill="#FFF"
      />
      <path
        d="M297 .05l-9.9 9.9L308.151 31 287.1 52.05l9.9 9.9L327.949 31z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="98.071"
        cy="123"
        fill="#FFF"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        transform="translate(43 68)"
      >
        <path
          d="M54.875 0c.334 0 .667.003 1 .009V10.01c-.333-.007-.666-.011-1-.011C30.091 10 10 30.091 10 54.875S30.091 99.75 54.875 99.75 99.75 79.659 99.75 54.875a44.665 44.665 0 00-7.124-24.272l8.273-5.624c5.598 8.601 8.851 18.869 8.851 29.896 0 30.307-24.568 54.875-54.875 54.875S0 85.182 0 54.875 24.568 0 54.875 0zm.15 22c.284 0 .567.004.85.01v10.005c-.283-.01-.566-.015-.85-.015C42.309 32 32 42.309 32 55.025S42.309 78.05 55.025 78.05 78.05 67.741 78.05 55.025c0-4.463-1.27-8.63-3.468-12.159l8.276-5.625a32.87 32.87 0 015.192 17.784c0 18.24-14.786 33.025-33.025 33.025C36.785 88.05 22 73.264 22 55.025 22 36.785 36.786 22 55.025 22z"
          fill-rule="nonzero"
        />
        <circle
          cx="54.875"
          cy="54.875"
          r="11.875"
        />
        <path
          d="M61 0v56H51V0z"
          fill-rule="nonzero"
        />
      </g>
    </g>
    <g
      transform="translate(0 314)"
    >
      <path
        d="M39 21c0 152.223 114.907 277.705 262.998 294.973l.001 14.092C146.166 312.725 25 180.97 25 21z"
        fill="#FF6EA9"
      />
      <path
        d="M20.475 31.525l-9.9 9.9-21.05-21.051-21.05 21.051-9.9-9.9L-10.475.576z"
        fill="#FF6EA9"
        fill-rule="nonzero"
      />
      <ellipse
        cx="129"
        cy="233"
        fill="#FF6EA9"
        rx="83"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M84.776 236.967l44.041 30.786.183.13.107-.076 44.239-30.754 2.534 1.78c2.764 1.948 2.825 6.074.184 8.118l-.184.136-44 31a5 5 0 01-5.76 0l-44-31c-2.827-1.991-2.827-6.261 0-8.253l2.656-1.867z"
        />
        <path
          d="M126.12 187.913l-44 31c-2.827 1.991-2.827 6.183 0 8.174l44 31a5 5 0 005.76 0l44-31c2.827-1.991 2.827-6.183 0-8.174l-44-31a5 5 0 00-5.76 0zm2.88 10.203L164.318 223 129 247.883 93.681 223 129 198.116z"
        />
        <path
          d="M145.4 216.904l8.394 5.909.167.23-22 16a5 5 0 01-5.685.137l-.197-.136-21.774-15.838 8.637-6.081 16.078 11.692 16.38-11.913z"
        />
        <path
          d="M134 178v52.361h-10V178z"
        />
      </g>
    </g>
    <g
      opacity="0.3"
      transform="translate(370 25)"
    >
      <path
        d="M.004.72C156.798 17.151 279 149.346 279 310h-14C265 157.093 149.057 31.167.003 14.8z"
        fill="#FFF"
      />
      <path
        d="M241.575 299.475l9.9-9.9 21.05 21.051 21.05-21.051 9.9 9.9-30.95 30.949z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="178.071"
        cy="98"
        fill="#FFF"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M178 47c-20.983 0-38 16.955-38 37.88l.003.482c.168 13.315 7.289 25.33 18.445 32.006l.093.055V133h38.993l-.001-15.624.25-.148C209.003 110.396 216 98.238 216 84.88 216 63.955 198.983 47 178 47zm0 10c15.468 0 28 12.486 28 27.88 0 9.829-5.145 18.77-13.418 23.806-.73.445-1.48.855-2.247 1.231l-2.801 1.372-.001 11.711h-18.992v-11.683l-2.812-1.369C156.18 105.304 150 95.645 150 84.88 150 69.486 162.532 57 178 57zm16.4 82.5l-5.772 10H167.37l-5.773-10H194.4z"
        />
        <path
          d="M179.236 69.692l-2.807 10.822h13.943l-5.183 21.793H174.91l2.804-11.793h-14.21l5.4-20.822h10.33z"
        />
      </g>
    </g>
    <g
      opacity="0.3"
    >
      <path
        d="M646.932 371.001C629.03 525.807 497.092 646 337 646v-14c152.358 0 278.004-113.951 295.833-260.997z"
        fill="#FFF"
      />
      <g
        fill-rule="nonzero"
        transform="translate(318 371)"
      >
        <use
          fill="#000"
          filter="url(#APPLY_svg__a)"
          xlink:href="#APPLY_svg__b"
        />
        <use
          fill="#FFF"
          xlink:href="#APPLY_svg__b"
        />
      </g>
      <g
        transform="translate(472 472)"
      >
        <ellipse
          cx="76.071"
          cy="75"
          fill="#FFF"
          rx="83.071"
          ry="82"
          stroke="#05192D"
          stroke-width="14"
        />
        <g
          fill="#05192D"
          fill-rule="nonzero"
        >
          <path
            d="M39 57.155l.038 34.711-10.006-5.777-.025-23.164L39 57.155zM54 49l.04 52-10 .008-.04-52zM113 92.423l-.039-35.868 10.006 5.777.025 24.321-9.993 5.77zM98 101l-.04-52 10-.008.04 52z"
          />
          <path
            d="M103 70v10H48V70z"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;

exports[`<ALPALoop /> renders a Assess logo 1`] = `
<svg
  viewBox="0 0 674 670"
>
  <defs>
    <filter
      filterUnits="objectBoundingBox"
      height="100%"
      id="ASSESS_svg__a"
      width="100%"
      x="0%"
      y="0%"
    >
      <feoffset
        in="SourceAlpha"
        result="shadowOffsetOuter1"
      />
      <fecolormatrix
        in="shadowOffsetOuter1"
        values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0"
      />
    </filter>
    <path
      d="M31.05 237.05l9.9 9.9L19.899 268l21.051 21.05-9.9 9.9L.101 268z"
      id="ASSESS_svg__b"
    />
  </defs>
  <g
    fill="none"
    fill-rule="evenodd"
  >
    <g
      transform="translate(27)"
    >
      <path
        d="M310 24v14C157.987 38 32.566 151.435 14.29 297.998H.185C18.54 143.675 150.253 24 310 24z"
        fill="#06BDFC"
      />
      <path
        d="M297 .05l-9.9 9.9L308.151 31 287.1 52.05l9.9 9.9L327.949 31z"
        fill="#06BDFC"
        fill-rule="nonzero"
      />
      <ellipse
        cx="98.071"
        cy="123"
        fill="#06BDFC"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        transform="translate(43 68)"
      >
        <path
          d="M54.875 0c.334 0 .667.003 1 .009V10.01c-.333-.007-.666-.011-1-.011C30.091 10 10 30.091 10 54.875S30.091 99.75 54.875 99.75 99.75 79.659 99.75 54.875a44.665 44.665 0 00-7.124-24.272l8.273-5.624c5.598 8.601 8.851 18.869 8.851 29.896 0 30.307-24.568 54.875-54.875 54.875S0 85.182 0 54.875 24.568 0 54.875 0zm.15 22c.284 0 .567.004.85.01v10.005c-.283-.01-.566-.015-.85-.015C42.309 32 32 42.309 32 55.025S42.309 78.05 55.025 78.05 78.05 67.741 78.05 55.025c0-4.463-1.27-8.63-3.468-12.159l8.276-5.625a32.87 32.87 0 015.192 17.784c0 18.24-14.786 33.025-33.025 33.025C36.785 88.05 22 73.264 22 55.025 22 36.785 36.786 22 55.025 22z"
          fill-rule="nonzero"
        />
        <circle
          cx="54.875"
          cy="54.875"
          r="11.875"
        />
        <path
          d="M61 0v56H51V0z"
          fill-rule="nonzero"
        />
      </g>
    </g>
    <g
      opacity="0.3"
      transform="translate(0 314)"
    >
      <path
        d="M39 21c0 152.223 114.907 277.705 262.998 294.973l.001 14.092C146.166 312.725 25 180.97 25 21z"
        fill="#FFF"
      />
      <path
        d="M20.475 31.525l-9.9 9.9-21.05-21.051-21.05 21.051-9.9-9.9L-10.475.576z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="129"
        cy="233"
        fill="#FFF"
        rx="83"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M84.776 236.967l44.041 30.786.183.13.107-.076 44.239-30.754 2.534 1.78c2.764 1.948 2.825 6.074.184 8.118l-.184.136-44 31a5 5 0 01-5.76 0l-44-31c-2.827-1.991-2.827-6.261 0-8.253l2.656-1.867z"
        />
        <path
          d="M126.12 187.913l-44 31c-2.827 1.991-2.827 6.183 0 8.174l44 31a5 5 0 005.76 0l44-31c2.827-1.991 2.827-6.183 0-8.174l-44-31a5 5 0 00-5.76 0zm2.88 10.203L164.318 223 129 247.883 93.681 223 129 198.116z"
        />
        <path
          d="M145.4 216.904l8.394 5.909.167.23-22 16a5 5 0 01-5.685.137l-.197-.136-21.774-15.838 8.637-6.081 16.078 11.692 16.38-11.913z"
        />
        <path
          d="M134 178v52.361h-10V178z"
        />
      </g>
    </g>
    <g
      opacity="0.3"
      transform="translate(370 25)"
    >
      <path
        d="M.004.72C156.798 17.151 279 149.346 279 310h-14C265 157.093 149.057 31.167.003 14.8z"
        fill="#FFF"
      />
      <path
        d="M241.575 299.475l9.9-9.9 21.05 21.051 21.05-21.051 9.9 9.9-30.95 30.949z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="178.071"
        cy="98"
        fill="#FFF"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M178 47c-20.983 0-38 16.955-38 37.88l.003.482c.168 13.315 7.289 25.33 18.445 32.006l.093.055V133h38.993l-.001-15.624.25-.148C209.003 110.396 216 98.238 216 84.88 216 63.955 198.983 47 178 47zm0 10c15.468 0 28 12.486 28 27.88 0 9.829-5.145 18.77-13.418 23.806-.73.445-1.48.855-2.247 1.231l-2.801 1.372-.001 11.711h-18.992v-11.683l-2.812-1.369C156.18 105.304 150 95.645 150 84.88 150 69.486 162.532 57 178 57zm16.4 82.5l-5.772 10H167.37l-5.773-10H194.4z"
        />
        <path
          d="M179.236 69.692l-2.807 10.822h13.943l-5.183 21.793H174.91l2.804-11.793h-14.21l5.4-20.822h10.33z"
        />
      </g>
    </g>
    <g
      opacity="0.3"
    >
      <path
        d="M646.932 371.001C629.03 525.807 497.092 646 337 646v-14c152.358 0 278.004-113.951 295.833-260.997z"
        fill="#FFF"
      />
      <g
        fill-rule="nonzero"
        transform="translate(318 371)"
      >
        <use
          fill="#000"
          filter="url(#ASSESS_svg__a)"
          xlink:href="#ASSESS_svg__b"
        />
        <use
          fill="#FFF"
          xlink:href="#ASSESS_svg__b"
        />
      </g>
      <g
        transform="translate(472 472)"
      >
        <ellipse
          cx="76.071"
          cy="75"
          fill="#FFF"
          rx="83.071"
          ry="82"
          stroke="#05192D"
          stroke-width="14"
        />
        <g
          fill="#05192D"
          fill-rule="nonzero"
        >
          <path
            d="M39 57.155l.038 34.711-10.006-5.777-.025-23.164L39 57.155zM54 49l.04 52-10 .008-.04-52zM113 92.423l-.039-35.868 10.006 5.777.025 24.321-9.993 5.77zM98 101l-.04-52 10-.008.04 52z"
          />
          <path
            d="M103 70v10H48V70z"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;

exports[`<ALPALoop /> renders a Learn logo 1`] = `
<svg
  viewBox="0 0 674 670"
>
  <defs>
    <filter
      filterUnits="objectBoundingBox"
      height="100%"
      id="LEARN_svg__a"
      width="100%"
      x="0%"
      y="0%"
    >
      <feoffset
        in="SourceAlpha"
        result="shadowOffsetOuter1"
      />
      <fecolormatrix
        in="shadowOffsetOuter1"
        values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0"
      />
    </filter>
    <path
      d="M31.05 237.05l9.9 9.9L19.899 268l21.051 21.05-9.9 9.9L.101 268z"
      id="LEARN_svg__b"
    />
  </defs>
  <g
    fill="none"
    fill-rule="evenodd"
  >
    <g
      opacity="0.3"
      transform="translate(27)"
    >
      <path
        d="M310 24v14C157.987 38 32.566 151.435 14.29 297.998H.185C18.54 143.675 150.253 24 310 24z"
        fill="#FFF"
      />
      <path
        d="M297 .05l-9.9 9.9L308.151 31 287.1 52.05l9.9 9.9L327.949 31z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="98.071"
        cy="123"
        fill="#FFF"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        transform="translate(43 68)"
      >
        <path
          d="M54.875 0c.334 0 .667.003 1 .009V10.01c-.333-.007-.666-.011-1-.011C30.091 10 10 30.091 10 54.875S30.091 99.75 54.875 99.75 99.75 79.659 99.75 54.875a44.665 44.665 0 00-7.124-24.272l8.273-5.624c5.598 8.601 8.851 18.869 8.851 29.896 0 30.307-24.568 54.875-54.875 54.875S0 85.182 0 54.875 24.568 0 54.875 0zm.15 22c.284 0 .567.004.85.01v10.005c-.283-.01-.566-.015-.85-.015C42.309 32 32 42.309 32 55.025S42.309 78.05 55.025 78.05 78.05 67.741 78.05 55.025c0-4.463-1.27-8.63-3.468-12.159l8.276-5.625a32.87 32.87 0 015.192 17.784c0 18.24-14.786 33.025-33.025 33.025C36.785 88.05 22 73.264 22 55.025 22 36.785 36.786 22 55.025 22z"
          fill-rule="nonzero"
        />
        <circle
          cx="54.875"
          cy="54.875"
          r="11.875"
        />
        <path
          d="M61 0v56H51V0z"
          fill-rule="nonzero"
        />
      </g>
    </g>
    <g
      opacity="0.3"
      transform="translate(0 314)"
    >
      <path
        d="M39 21c0 152.223 114.907 277.705 262.998 294.973l.001 14.092C146.166 312.725 25 180.97 25 21z"
        fill="#FFF"
      />
      <path
        d="M20.475 31.525l-9.9 9.9-21.05-21.051-21.05 21.051-9.9-9.9L-10.475.576z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="129"
        cy="233"
        fill="#FFF"
        rx="83"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M84.776 236.967l44.041 30.786.183.13.107-.076 44.239-30.754 2.534 1.78c2.764 1.948 2.825 6.074.184 8.118l-.184.136-44 31a5 5 0 01-5.76 0l-44-31c-2.827-1.991-2.827-6.261 0-8.253l2.656-1.867z"
        />
        <path
          d="M126.12 187.913l-44 31c-2.827 1.991-2.827 6.183 0 8.174l44 31a5 5 0 005.76 0l44-31c2.827-1.991 2.827-6.183 0-8.174l-44-31a5 5 0 00-5.76 0zm2.88 10.203L164.318 223 129 247.883 93.681 223 129 198.116z"
        />
        <path
          d="M145.4 216.904l8.394 5.909.167.23-22 16a5 5 0 01-5.685.137l-.197-.136-21.774-15.838 8.637-6.081 16.078 11.692 16.38-11.913z"
        />
        <path
          d="M134 178v52.361h-10V178z"
        />
      </g>
    </g>
    <g
      transform="translate(370 25)"
    >
      <path
        d="M.004.72C156.798 17.151 279 149.346 279 310h-14C265 157.093 149.057 31.167.003 14.8z"
        fill="#03EF62"
      />
      <path
        d="M241.575 299.475l9.9-9.9 21.05 21.051 21.05-21.051 9.9 9.9-30.95 30.949z"
        fill="#03EF62"
        fill-rule="nonzero"
      />
      <ellipse
        cx="178.071"
        cy="98"
        fill="#03EF62"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M178 47c-20.983 0-38 16.955-38 37.88l.003.482c.168 13.315 7.289 25.33 18.445 32.006l.093.055V133h38.993l-.001-15.624.25-.148C209.003 110.396 216 98.238 216 84.88 216 63.955 198.983 47 178 47zm0 10c15.468 0 28 12.486 28 27.88 0 9.829-5.145 18.77-13.418 23.806-.73.445-1.48.855-2.247 1.231l-2.801 1.372-.001 11.711h-18.992v-11.683l-2.812-1.369C156.18 105.304 150 95.645 150 84.88 150 69.486 162.532 57 178 57zm16.4 82.5l-5.772 10H167.37l-5.773-10H194.4z"
        />
        <path
          d="M179.236 69.692l-2.807 10.822h13.943l-5.183 21.793H174.91l2.804-11.793h-14.21l5.4-20.822h10.33z"
        />
      </g>
    </g>
    <g
      opacity="0.3"
    >
      <path
        d="M646.932 371.001C629.03 525.807 497.092 646 337 646v-14c152.358 0 278.004-113.951 295.833-260.997z"
        fill="#FFF"
      />
      <g
        fill-rule="nonzero"
        transform="translate(318 371)"
      >
        <use
          fill="#000"
          filter="url(#LEARN_svg__a)"
          xlink:href="#LEARN_svg__b"
        />
        <use
          fill="#FFF"
          xlink:href="#LEARN_svg__b"
        />
      </g>
      <g
        transform="translate(472 472)"
      >
        <ellipse
          cx="76.071"
          cy="75"
          fill="#FFF"
          rx="83.071"
          ry="82"
          stroke="#05192D"
          stroke-width="14"
        />
        <g
          fill="#05192D"
          fill-rule="nonzero"
        >
          <path
            d="M39 57.155l.038 34.711-10.006-5.777-.025-23.164L39 57.155zM54 49l.04 52-10 .008-.04-52zM113 92.423l-.039-35.868 10.006 5.777.025 24.321-9.993 5.77zM98 101l-.04-52 10-.008.04 52z"
          />
          <path
            d="M103 70v10H48V70z"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;

exports[`<ALPALoop /> renders a Practice logo 1`] = `
<svg
  viewBox="0 0 674 670"
>
  <defs>
    <filter
      filterUnits="objectBoundingBox"
      height="100%"
      id="PRACTICE_svg__a"
      width="100%"
      x="0%"
      y="0%"
    >
      <feoffset
        in="SourceAlpha"
        result="shadowOffsetOuter1"
      />
      <fecolormatrix
        in="shadowOffsetOuter1"
        values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.5 0"
      />
    </filter>
    <path
      d="M31.05 237.05l9.9 9.9L19.899 268l21.051 21.05-9.9 9.9L.101 268z"
      id="PRACTICE_svg__b"
    />
  </defs>
  <g
    fill="none"
    fill-rule="evenodd"
  >
    <g
      opacity="0.3"
      transform="translate(27)"
    >
      <path
        d="M310 24v14C157.987 38 32.566 151.435 14.29 297.998H.185C18.54 143.675 150.253 24 310 24z"
        fill="#FFF"
      />
      <path
        d="M297 .05l-9.9 9.9L308.151 31 287.1 52.05l9.9 9.9L327.949 31z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="98.071"
        cy="123"
        fill="#FFF"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        transform="translate(43 68)"
      >
        <path
          d="M54.875 0c.334 0 .667.003 1 .009V10.01c-.333-.007-.666-.011-1-.011C30.091 10 10 30.091 10 54.875S30.091 99.75 54.875 99.75 99.75 79.659 99.75 54.875a44.665 44.665 0 00-7.124-24.272l8.273-5.624c5.598 8.601 8.851 18.869 8.851 29.896 0 30.307-24.568 54.875-54.875 54.875S0 85.182 0 54.875 24.568 0 54.875 0zm.15 22c.284 0 .567.004.85.01v10.005c-.283-.01-.566-.015-.85-.015C42.309 32 32 42.309 32 55.025S42.309 78.05 55.025 78.05 78.05 67.741 78.05 55.025c0-4.463-1.27-8.63-3.468-12.159l8.276-5.625a32.87 32.87 0 015.192 17.784c0 18.24-14.786 33.025-33.025 33.025C36.785 88.05 22 73.264 22 55.025 22 36.785 36.786 22 55.025 22z"
          fill-rule="nonzero"
        />
        <circle
          cx="54.875"
          cy="54.875"
          r="11.875"
        />
        <path
          d="M61 0v56H51V0z"
          fill-rule="nonzero"
        />
      </g>
    </g>
    <g
      opacity="0.3"
      transform="translate(0 314)"
    >
      <path
        d="M39 21c0 152.223 114.907 277.705 262.998 294.973l.001 14.092C146.166 312.725 25 180.97 25 21z"
        fill="#FFF"
      />
      <path
        d="M20.475 31.525l-9.9 9.9-21.05-21.051-21.05 21.051-9.9-9.9L-10.475.576z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="129"
        cy="233"
        fill="#FFF"
        rx="83"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M84.776 236.967l44.041 30.786.183.13.107-.076 44.239-30.754 2.534 1.78c2.764 1.948 2.825 6.074.184 8.118l-.184.136-44 31a5 5 0 01-5.76 0l-44-31c-2.827-1.991-2.827-6.261 0-8.253l2.656-1.867z"
        />
        <path
          d="M126.12 187.913l-44 31c-2.827 1.991-2.827 6.183 0 8.174l44 31a5 5 0 005.76 0l44-31c2.827-1.991 2.827-6.183 0-8.174l-44-31a5 5 0 00-5.76 0zm2.88 10.203L164.318 223 129 247.883 93.681 223 129 198.116z"
        />
        <path
          d="M145.4 216.904l8.394 5.909.167.23-22 16a5 5 0 01-5.685.137l-.197-.136-21.774-15.838 8.637-6.081 16.078 11.692 16.38-11.913z"
        />
        <path
          d="M134 178v52.361h-10V178z"
        />
      </g>
    </g>
    <g
      opacity="0.3"
      transform="translate(370 25)"
    >
      <path
        d="M.004.72C156.798 17.151 279 149.346 279 310h-14C265 157.093 149.057 31.167.003 14.8z"
        fill="#FFF"
      />
      <path
        d="M241.575 299.475l9.9-9.9 21.05 21.051 21.05-21.051 9.9 9.9-30.95 30.949z"
        fill="#FFF"
        fill-rule="nonzero"
      />
      <ellipse
        cx="178.071"
        cy="98"
        fill="#FFF"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M178 47c-20.983 0-38 16.955-38 37.88l.003.482c.168 13.315 7.289 25.33 18.445 32.006l.093.055V133h38.993l-.001-15.624.25-.148C209.003 110.396 216 98.238 216 84.88 216 63.955 198.983 47 178 47zm0 10c15.468 0 28 12.486 28 27.88 0 9.829-5.145 18.77-13.418 23.806-.73.445-1.48.855-2.247 1.231l-2.801 1.372-.001 11.711h-18.992v-11.683l-2.812-1.369C156.18 105.304 150 95.645 150 84.88 150 69.486 162.532 57 178 57zm16.4 82.5l-5.772 10H167.37l-5.773-10H194.4z"
        />
        <path
          d="M179.236 69.692l-2.807 10.822h13.943l-5.183 21.793H174.91l2.804-11.793h-14.21l5.4-20.822h10.33z"
        />
      </g>
    </g>
    <path
      d="M646.932 371.001C629.03 525.807 497.092 646 337 646v-14c152.358 0 278.004-113.951 295.833-260.997z"
      fill="#FF931E"
    />
    <g
      fill-rule="nonzero"
      transform="translate(318 371)"
    >
      <use
        fill="#000"
        filter="url(#PRACTICE_svg__a)"
        xlink:href="#PRACTICE_svg__b"
      />
      <use
        fill="#FF931E"
        xlink:href="#PRACTICE_svg__b"
      />
    </g>
    <g
      transform="translate(472 472)"
    >
      <ellipse
        cx="76.071"
        cy="75"
        fill="#FF931E"
        rx="83.071"
        ry="82"
        stroke="#05192D"
        stroke-width="14"
      />
      <g
        fill="#05192D"
        fill-rule="nonzero"
      >
        <path
          d="M39 57.155l.038 34.711-10.006-5.777-.025-23.164L39 57.155zM54 49l.04 52-10 .008-.04-52zM113 92.423l-.039-35.868 10.006 5.777.025 24.321-9.993 5.77zM98 101l-.04-52 10-.008.04 52z"
        />
        <path
          d="M103 70v10H48V70z"
        />
      </g>
    </g>
  </g>
</svg>
`;
